features/sctp/test/Gopkg.toml

```diff
@@ -0,0 +1,55 @@
+# Gopkg.toml example
```

Comment:
Is there a reason not to use go modules for dependency management? There's a lot of momentum around using modules now and it looks like the way to go for new projects when at all possible. Both kubevirt and ocs switched to go modules for example.

Reply:
No actual reason beyond my laziness after a couple of failed attempts.
Dep worked out of the box, where go mod did not. Will try to make it work and submit an additional pr.

Reply:
fyi, this will require golang 1.11 and greater. 

I know jump to yet another dependency manager is a pain. After the learning curve, working with go modules has proven to be more efficient for us in the past than dep or glide.

Reply:
Yes I know, I made some work around go modules in the past.
Need to understand better how to deal with k8s dependencies and if it's still the case that I need to specify each version manually, but I'll make it work somehow.

Reply:
if you need some help with go modules you can ping me :)

Reply:
@cynepco3hahue thanks! I think I just finished. Will ask for your review afterwards.